packages/vite/src/modes/global/build.ts

```diff
@@ -279,8 +279,12 @@ export function GlobalModeBuildPlugin(ctx: UnocssPluginContext<VitePluginConfig>
           // to replace on current build pipeline, we can skip the warning.
           if (replaced)
             return
-          const msg = '[unocss] Entry module not found. Did you add `import \'uno.css\'` in your main entry?'
-          this.warn(msg)
+
+          if (!(await getConfig() as VitePluginConfig).disableEntryCheck) {
+            const msg = '[unocss] Entry module not found. Did you add `import \'uno.css\'` in your main entry?'
+            this.warn(msg)
```

Comment:
```suggestion
            this.warn('[unocss] Entry module not found. Did you add `import \'uno.css\'` in your main entry?')
```

Reply:
Message has been extracted to a shared constant as it's been used three times within two files.